docs/source/api/safe/twobody/propagation.rst

```diff
@@ -0,0 +1,5 @@
+Propagation
+===========
+
+.. automodule:: poliastro.twobody.propagation
```

Comment:
Perhaps it's a good moment to add even a one-line docstring to `kepler` and `mean_motion`...